src/Core/src/Handlers/DatePicker/DatePickerHandler.iOS.cs

```diff
@@ -8,27 +8,21 @@ namespace Microsoft.Maui.Handlers
 #if IOS && !MACCATALYST
 	public partial class DatePickerHandler : ViewHandler<IDatePicker, MauiDatePicker>
 	{
-		UIDatePicker? _picker;
-
 		protected override MauiDatePicker CreatePlatformView()
 		{
 			MauiDatePicker platformDatePicker = new MauiDatePicker();
-
-			_picker = new UIDatePicker { Mode = UIDatePickerMode.Date, TimeZone = new NSTimeZone("UTC") };
-
-			if (OperatingSystem.IsIOSVersionAtLeast(13, 4))
-			{
-				_picker.PreferredDatePickerStyle = UIDatePickerStyle.Wheels;
-			}
-
-			platformDatePicker.InputView = _picker;
-			platformDatePicker.InputAccessoryView = new MauiDoneAccessoryView(() =>
+			var accessoryView = new MauiDoneAccessoryView();
+			accessoryView.DoneClicked += (_, _) =>
```

Comment:
This doesn't leak. No idea why. It's capturing "this" and it all collects just fine. The MauiTimePicker however does leak given this exact same code